aire-core/src/main/frontend/components/aire/aire-media-card.ts

```diff
@@ -0,0 +1,24 @@
+import {
```

Comment:
We could derive from a common base-class instead of litelement

Reply:
...unnecessary for now since neither JS element has any behaviors, but maybe in the future.